real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/config.java

```diff
@@ -38,23 +38,23 @@
 	public static boolean AUTONOMUS_DEFAULT = false;
 
 	// sensor default settings (true for on false for off)
-	public static boolean DRIVE_DEFAULT = true;
-	public static boolean IMU_DEFAULT = true;
-	public static boolean GPS_DEFAULT = true;
-	public static boolean ENCODER_DEFAULT = true;
-	public static boolean VISION_SYSTEM_DEFAULT = true;
+	public static  boolean DRIVE_DEFAULT = true;
+	public static  boolean IMU_DEFAULT = true;
+	public static  boolean GPS_DEFAULT = true;
+	public static  boolean ENCODER_DEFAULT = true;
+	public static  boolean VISION_SYSTEM_DEFAULT = true;
 	
 	//sensor com ports
-	public static String COM_PORT_IMU = "COM6";
-	public static String COM_PORT_GPS_INTEGRATED = "COM5";
-	public static String COM_PORT_GPS_STANDALONE = "COM8";
-	public static String COM_PORT_ENCODER = "COM3";
-	public static String COM_PORT_DRIVE_CONTROL = "COM9";
+	public static  String COM_PORT_IMU = "COM6";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1268275770/issues/source?bid=2938665&fileBranchId=2939970#l48)